include/openssl/evp.h

```diff
@@ -1678,11 +1678,15 @@ int EVP_PKEY_get_raw_private_key(const EVP_PKEY *pkey, unsigned char *priv,
 int EVP_PKEY_get_raw_public_key(const EVP_PKEY *pkey, unsigned char *pub,
                                 size_t *len);
 
+# ifndef OPENSSL_NO_DEPRECATED_3_0
+OSSL_DEPRECATEDIN_3_0
```

Comment:
This function should be removed, not deprecated.

Reply:
`EVP_PKEY_new_CMAC_key_ex`, that is...

Reply:
I removed `EVP_PKEY_new_CMAC_key_ex` and its uses in tests.